components/locid/examples/canonicalize_locales.rs

```diff
@@ -0,0 +1,47 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/master/LICENSE ).
+// A sample application which takes a comma separated list of locales,
+// makes them canonical and serializes the list back into a comma separated list.
+
+use std::env;
+
+use icu_locid::Locale;
+
+const DEFAULT_INPUT: &str = "sr-cyrL-rS, es-mx, und-arab-u-ca-Buddhist";
+
+fn canonicalize_locales(input: &str) -> String {
+    // Split input string and canonicalize each locale identifier.
```

Comment:
This is performing syntax canonicalization, but not data-based subtag alias canonlicalization.  The full-size canonicalization is tracked in #218.  Can you change the file name and function name to reflect that?

Reply:
Good catch, will do.